src/Header/Cookie.php

```diff
@@ -55,10 +62,14 @@ public static function fromString($headerLine)
 
         $nvPairs = preg_split('#;\s*#', $value);
 
+        if (false === $nvPairs) {
+            throw new Exception\RuntimeException('Malformed Cookie header found');
+        }
```

Comment:
This is new behavior and requires a corresponding test.

Reply:
Removed and set `$nvPairs` as `string[]`. The only waay it could return ` false` is when pattern is not valid, and this can be covered in tests.